sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/notifications/signaling/CommunicationSignalingClient.java

```diff
@@ -221,6 +225,8 @@ private void start(Context context, ISkypetokenProvider skypetokenProvider, Stri
         }
 
         CloudType cloudType = NotificationUtils.getUserCloudTypeFromSkypeToken(skypeUserToken);
+        String resourceLocation = NotificationUtils.decodeResourceLocationFromJwtToken(skypeUserToken);
+        boolean isEUDBCountry = resourceLocation != null && countriesEUDB.contains(resourceLocation);
```

Comment:
Will the resource location in the JWT always be lowercase? Maybe we should do something like the following if we're not sure:
```suggestion
        boolean isEUDBCountry = resourceLocation != null && countriesEUDB.contains(resourceLocation.toLowerCase());
```